.github/workflows/test-wheel-linux.yml

```
@@ -318,14 +318,13 @@ jobs:
           pushd "${CUDA_BINDINGS_ARTIFACTS_DIR}"
           if [[ "${{ matrix.LOCAL_CTK }}" == 1 ]]; then
             ls $CUDA_PATH
-            pip install *.whl
+            pip install *.whl[test]
```

This is one of the annoying bits: In order to use the bracket syntax, we need to expand the wildcard first (following what's done below)
```suggestion
            pip install $(ls *.whl)[test]
```